sql/database.sql

```diff
@@ -157,7 +157,7 @@ CREATE TABLE `background_services` (
 INSERT INTO `background_services` (`name`, `title`, `execute_interval`, `function`, `require_once`, `sort_order`) VALUES
 ('phimail', 'phiMail Direct Messaging Service', 5, 'phimail_check', '/library/direct_message_check.inc', 100);
 INSERT INTO `background_services` (`name`, `title`, `active`, `running`, `next_run`, `execute_interval`, `function`, `require_once`, `sort_order`) VALUES
-('MedEx', 'MedEx Messaging Service', 0, 0, '2017-05-09 17:39:10', 0, 'start_MedEx', '/library/MedEx/medex_background.php', 100);
+('MedEx', 'MedEx Messaging Service', 0, 0, '2017-05-09 17:39:10', 0, 'start_MedEx', '/library/MedEx/MedEx_background.php', 100);
```

Comment:
hi @ophthal ,
There are 2 ways to go about this.
1. Do what you did and then you need to also fix this in the sql upgrade script. (don't worry about the patch mechanism; I can add it to that).
2. Instead change the filename of the script instead to medex_background.php :) (note this will actually be easier for you and also easier to fix in the patch)

I will leave the decision up to you.
-brady